examples/contrib/cifar10/main.py

```diff
@@ -380,9 +380,6 @@ def evaluate_step(engine: Engine, batch):
     for name, metric in metrics.items():
         metric.attach(evaluator, name)
 
-    if idist.get_rank() == 0 and (not config["with_clearml"]):
```

Comment:
Why did you this pbar?

Reply:
I does not look helpful and also adds useless lines if logs are written into a file